lib/lvol/lvol.c

```diff
@@ -1536,6 +1552,14 @@ spdk_lvol_create_snapshot_with_xattrs(struct spdk_lvol *origlvol, const char *sn
 		return;
 	}
 
+	for (i = 0; i < xattrs_external_num; i += 2) {
```

Comment:
Why `i+=2` in the for loop?
Can you add a comment in the above of the for loop?

Reply:
Done, thanks for the suggestion, it was not intuitive.